src/modules/musig/main_impl.h

```diff
@@ -129,6 +131,7 @@ int secp256k1_musig_pubkey_combine(const secp256k1_context* ctx, secp256k1_scrat
         return 0;
     }
     if (!secp256k1_ecmult_multi_var(&ctx->error_callback, &ctx->ecmult_ctx, scratch, &pkj, NULL, secp256k1_musig_pubkey_combine_callback, (void *) &ecmult_data, n_pubkeys)) {
+        /* The current implementation of ecmult_multi_var makes this code unreachable with tests. */
```

Comment:
If you want, you could try playing around with https://gcovr.com/en/stable/guide.html#exclusion-markers . (But this can be done later)

Reply:
let's do this later